Source/Fuse.Platform/iOS/SystemUI.uno

```diff
@@ -161,74 +216,68 @@ namespace Fuse.Platform
 
 		@}
 
-		static void OnWillResize(SystemUIWillResizeEventArgs args)
+		static void OnWillResize()
 		{
-			if (args.ID==SystemUIID.TopFrame) {
-				EventHandler<SystemUIWillResizeEventArgs> handler = TopFrameWillResize;
-				if (handler != null)
-					handler(null, args);
-			} else {
-				BottomFrame = args.EndFrame;
-				EventHandler<SystemUIWillResizeEventArgs> handler = BottomFrameWillResize;
-				if (handler != null)
-					handler(null, args);
-			}
+			if (MarginsChanged != null)
+				MarginsChanged();
 		}
 
 		static Rect GetStatusBarFrame()
 		{
-			var density = Uno.Platform.Displays.MainDisplay.Density;
+			var density = 1;//Uno.Platform.Displays.MainDisplay.Density;
```

Comment:
This seems bad. :)

Reply:
The density stuff was intentional as I changed the contract on SystemUI to return units in the device unit size. You  need access to a `Viewport` objecct to convert correctly, and that is done inside `window()` now. There are no other users of SystemUI.  Plus, most of the old interfaces to it were removed.

I don't think it should cause any ill-effects.

Reply:
That sounds good; it looked a bit suspect due to the comments though.

Reply:
Yeah, I'll cleanup the comments.